.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+### What
+<!-- Explain what you changed and provide a list of changed page names. -->
+
+### Why
+<!-- Explain why this change is necessary and how it benefits users. -->
+
+### Screenshots
+<!-- Optional. Show additions to the sidebar or new formatting. -->
+
+----------
+
+### Merge Checklist
+_If items do not apply to your changes, add (N/A) and mark them as complete._
+
+#### Pull Request
+- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
+- [ ] Description links to related pull requests or issues, if any.
+
+#### Content
+- [ ] Redirects have been added for moved, renamed, or deleted pages.
```

Comment:
Just a callout:

Sub project redirects are not yet supported. They need be handled in https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js

We do have an Asana task for this though!

Reply:
OHHH okay thank you!! let me add some further explanation :)